src/tests/ftest/util/soak_utils.py

```diff
@@ -320,20 +320,22 @@ def wait_for_pool_rebuild(self, pool, name):
     """
     rebuild_status = False
     self.log.info("<<Wait for %s rebuild on %s>> at %s", name, pool.identifier, time.ctime())
-    self.dmg_command.server_set_logmasks("DEBUG", raise_exception=False)
+    if self.enable_rebuild_logmasks:
+        self.dmg_command.server_set_logmasks("DEBUG", raise_exception=False)
     try:
         # # Wait for rebuild to start
         # pool.wait_for_rebuild_to_start()
         # Wait for rebuild to complete
         pool.wait_for_rebuild_to_end()
         rebuild_status = True
     except DaosTestError as error:
-        self.log.error(f"<<<FAILED:{name} rebuild timed out: {error}", exc_info=error)
+        self.log.error(f"<<<FAILED: {name} rebuild timed out: {error}", exc_info=error)
         rebuild_status = False
     except TestFail as error1:
         self.log.error(
-            f"<<<FAILED:{name} rebuild failed due to test issue: {error1}", exc_info=error1)
-    self.dmg_command.server_set_logmasks(raise_exception=False)
+            f"<<<FAILED: {name} rebuild failed due to test issue: {error1}", exc_info=error1)
+    if self.enable_rebuild_logmasks:
+        self.dmg_command.server_set_logmasks(raise_exception=False)
```

Comment:
To ensure we put this setting back if we hit a test timeout while waiting for rebuild:
```suggestion
    finally:
        if self.enable_rebuild_logmasks:
            self.dmg_command.server_set_logmasks(raise_exception=False)
```